src/client/Engine.cpp

```diff
@@ -2,9 +2,16 @@
 #include "Engine.h"
 
 #include "window/GlfwWindow.h"
+#include "graphics/GraphicsConfig.h"
 
-IWindow &Engine::getWindow(int width, int height, const std::string &title)
+IWindow &Engine::getWindow(const std::string &title)
```

Comment:
if we use a config for width and height, then why don't we use it for title too?

Reply:
> if we use a config for width and height, then why don't we use it for title too?

I don't think it makes sense to allow the user to change the name of the window via config. The name of the window should have a strictly defined name of the game

Reply:
> @Kelvium
> 
> > if we use a config for width and height, then why don't we use it for title too?
> 
> I don't think it makes sense to allow the user to change the name of the window via config. The name of the window should have a strictly defined name of the game

that makes sense. but i think setting the window size in config is not convenient. i'd prefer to leave a "TODO: provide a convenient way of changing window size" where the window is created as this is not important and we can implement it when we got to the point where we can make "non-important" tagged bugs/things/etc. :)

Reply:
> > @Kelvium
> > > if we use a config for width and height, then why don't we use it for title too?
> > 
> > 
> > I don't think it makes sense to allow the user to change the name of the window via config. The name of the window should have a strictly defined name of the game
> 
> that makes sense. but i think setting the window size in config is not convenient. i'd prefer to leave a "TODO: provide a convenient way of changing window size" where the window is created as this is not important and we can implement it when we got to the point where we can make "non-important" tagged bugs/things/etc. :)

I thought about it. And I think that the size of the window should be left in the config to save the settings. I thought that the config file will be used to save settings when the user changes something in the game settings